youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import get_element_by_class, determine_ext, clean_html, KNOWN_EXTENSIONS
+import re
+
+
+class WikimediaIE(InfoExtractor):
+    _NETRC_MACHINE = 'wikimediaorg'
+    IE_NAME = 'wikimedia.org'
+    _API_BASE_URL = 'https://commons.wikimedia.org/'
+    _VALID_URL = r'https://commons.wikimedia.org/wiki/File:(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://commons.wikimedia.org/wiki/File:Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS.webm',
+        'info_dict': {
+            'description': 'md5:D6F4C7BF1C0DB1EAE80371B1F93EA85E',
+            'ext': 'webm',
+            'id': 'Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS',
+            'title': 'Die Temperaturkurve der Erde (ZDF, Terra X) 720p HD 50FPS.webm - Wikimedia Commons',
+            'license': 'This file is licensed under the Creative Commons Attribution 4.0 International license.',
+            'author': 'ZDF/Terra X/Gruppe 5/Luise Wagner, Jonas Sichert, Andreas Hougardy',
+            'subtitles': {'de': [
+                {'ext': 'srt',
+                 'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp&title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp&lang=de&amp&trackformat=srt'}],
+                'en-gb': [
+                    {'ext': 'srt',
+                     'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp&title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp&lang=en-gb&amp&trackformat=srt'}],
+                'nl': [
+                    {'ext': 'srt',
+                     'url': 'https://commons.wikimedia.org/w/api.php?action=timedtext&amp&title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS.webm&amp&lang=nl&amp&trackformat=srt'}
+                ]}
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        ext = determine_ext(url)
+        if not ext.lower() in KNOWN_EXTENSIONS:
+            raise Exception("invalid video url")
+
+        webpage = self._download_webpage(url, video_id)
+        self.report_extraction(video_id)
+        video_url = self._html_search_regex(r'<source [^>]*src="([^"]+)"', webpage,
+                                            u'video URL')
+        licenze = self._html_search_regex(r'\bThis\s*(.*?)\s*license\b', webpage, u'video license')
+        licenze = "This " + licenze + " license."
+
+        description = get_element_by_class('description', webpage)
+        author = self._html_search_regex(r'<td>([^\<]*?)<\/td>', str(webpage), u"video author")
```

Comment:
Shouldn't crash if absent. Also:
* don't use `str()`. `webpage` will already be a `compat_str` and so the right type to be passed to `re` search functions in either Py2 or Py3
* relax the RE
* since you fixed the `unicode_literals` import, remove all `u` from explicit `u'...'`, here and anywhere else.
```suggestion
        author = self._html_search_regex(r'<td\b[^>]*>\s*([^<]*?)\s*</td>', webpage, 'video author', default=None)
```
If you want a diagnostic for missing author:

```suggestion
        author = self._html_search_regex(r'<td\b[^>]*>\s*([^<]*?)\s*</td>', webpage, 'video author', fatal=False)
```